app/containers/MessageBox/index.js

```diff
@@ -797,13 +857,38 @@ class MessageBox extends Component {
 		}
 	}
 
+	changeComposerState = () => {
+		const { fullScreenUp } = this.state;
+
+		if (fullScreenUp) {
+			this.setState({ translateY: BOTTOM });
+		} else {
+			this.setState({ translateY: TOP });
+		}
+
+		this.setState({ fullScreenUp: !fullScreenUp });
+	}
+
+	renderTopButton = () => {
+		const { fullScreenUp } = this.state;
+		const { theme, editing } = this.props;
+		return (
+			<TouchableOpacity onPress={() => this.changeComposerState()} style={[stylez.topButton, { backgroundColor: themes[theme].messageboxBackground }, editing && { backgroundColor: themes[theme].chatComponentBackground }]}>
```

Comment:
The value for `style` is quite long and complicated. Could it be extracted to a variable?

Reply:
Can be extracted, I'll do it